project/OSGi.scala

```diff
@@ -91,8 +93,16 @@ object OSGi {
     exports(
       packages = Seq(
         "akka.stream.*",
-        "com.typesafe.sslconfig.akka.*"),
-      imports = Seq(scalaJava8CompatImport(), scalaParsingCombinatorImport(), sslConfigCoreImport()))
+        "com.typesafe.sslconfig.akka.*"
+      ),
+      imports = Seq(
+        scalaJava8CompatImport(),
+        scalaParsingCombinatorImport(),
+        sslConfigCoreImport("com.typesafe.sslconfig.ssl.*"),
+        sslConfigCoreImport("com.typesafe.sslconfig.util.*"),
+        "!com.typesafe.sslconfig.akka.*"
```

Comment:
I guess this change is correct

Reply:
I have now rebased and pushed. I can test within the next couple of days if the snapshot works for me in my project.